flang/lib/Lower/ConvertExpr.cpp

```diff
@@ -978,7 +978,40 @@ class ExprLowering {
   Fortran::lower::ExValue gen(const Fortran::evaluate::ArrayRef &aref) {
```

Comment:
What is `ExValue`?  Extract value?  This function might could use a rename?

Reply:
It's just an alias for the `ExtendedValue` class added by Eric in PR #75 to return all mlir::Value that describe array and character expressions.
https://github.com/flang-compiler/f18-llvm-project/blob/388d24bd0c33740c198318fe599f9d172d4c11a8/flang/include/flang/Lower/Support/BoxValue.h#L200